services/Mail.js

```diff
@@ -263,6 +263,25 @@ class Mail {
       }
     );
   }
+
+  /**
+   * @description Send appreciation email after successful donation
+   */
+  static sendThankYouForDonation(email, amount) {
+    this._sendEmail(
+      {
+        email,
+        subject: "Thank You For Your Support",
+        template: "thank-you-for-donating",
+        params: {
+          amount
+        }
+      },
+      (err, data) => {
+        if (err) console.log(err);
```

Comment:
@Chiemezuo, we should be using our `Logger` not `console.log`, I'll make the changes though - just wanted to let you know.

Reply:
Thank you. Sorry I miss it from time to time.